### PR TITLE
Update database config messaging

### LINKED ISSUE #1280: Tell users to configure their database after mix phoenix.server
Today, we tell users after phoenix.install to run `mix ecto.create`. However, often their database is not properly configured which puts them far from the rewards of seeing their phoenix app running. Because we don't need the repo on the home, we should tell users to run Ecto at the end:

```
Before moving on, configure your database in config/dev.exs and run:

    mix ecto.create

```

/cc @napcs

### PR DESCRIPTION
**Pull request by doomspork**
Alright @josevalim, this should be more like what you expected.  Sorry for the confusion earlier.  :grinning: 

```shell
We are all set! Run your Phoenix application:

    $ cd example_test
    $ mix phoenix.server

You can also run your app inside IEx (Interactive Elixir) as:

    $ iex -S mix phoenix.server

Before moving on, configure your database in config/dev.exs and run:

    $ mix ecto.create
```

Closes #1280

### CODE REVIEW

**Comment by josevalim** [comment]
We have tests inside the installer directory and they fail. You can run them by uninstalling any phoenix local archive and running `mix test`. Can you please check? Thank you!

**Comment by doomspork** [comment]
:facepalm:  I didn't add the test changes.  Done.

File: installer/test/phoenix_new_test.exs [hunk 1]
```diff
@@ -105,9 +105,11 @@ defmodule Mix.Tasks.Phoenix.NewTest do
       # Instructions
       assert_received {:mix_shell, :info, ["\nWe are all set!" <> _ = msg]}
       assert msg =~ "$ cd photo_blog"
-      assert msg =~ "$ mix ecto.create"
       assert msg =~ "$ mix phoenix.server"
 
+      assert_received {:mix_shell, :info, ["Before moving on," <> _ = msg]}
+       assert msg =~ "$ mix ecto.create"
```
Comment (josevalim) [inline installer/test/phoenix_new_test.exs]: Oops, wrong indent.

**Comment by josevalim** [comment]
One last commt and we are good to go!

**Comment by doomspork** [comment]
Updated, thanks @josevalim :smile:

**Pull request merged**